packages/core/src/codewhisperer/util/authUtil.ts

```diff
@@ -283,9 +283,9 @@ export class AuthUtil {
         return this.conn.getCredentials()
     }
 
-    public isConnectionValid(): boolean {
+    public isConnectionValid(silent: boolean = false): boolean {
         const connectionValid = this.conn !== undefined && !this.secondaryAuth.isConnectionExpired
-        if (connectionValid === false) {
+        if (!silent) {
```

Comment:
actually I prefer to remove this logging here, but want to align with folks here

Reply:
You can use `onceChanged` to only log if the previous value is different: https://github.com/aws/aws-toolkit-vscode/blob/2c4ea6516c5249c048b301d5826f4b0cfe16bf0e/packages/core/src/shared/utilities/functionUtils.ts#L54

Reply:
Example: https://github.com/aws/aws-toolkit-vscode/blob/e5fdf4109207f801a9b91a4d162ef1fb8d1bffd1/packages/core/src/shared/sam/cli/samCliSettings.ts#L60

Reply:
You can use `onceChanged` to only log if the previous value is different: https://github.com/aws/aws-toolkit-vscode/blob/e5fdf4109207f801a9b91a4d162ef1fb8d1bffd1/packages/core/src/shared/sam/cli/samCliSettings.ts#L60

Reply:
gotcha, that would be a better solution here, thanks Justin